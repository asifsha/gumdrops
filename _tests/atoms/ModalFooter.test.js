/* globals mount */
import React from 'react';
import ModalFooter from '../../components/atoms/ModalFooter';

const defaultProps = {
    className: 'modal-class',
    style: { width: '100px' }
};

describe('Expect <ModalFooter>', () => {
    it('to render', () => {
        const wrapper = mount(
            <ModalFooter {...defaultProps}>
                <div>My modal content</div>
            </ModalFooter>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
