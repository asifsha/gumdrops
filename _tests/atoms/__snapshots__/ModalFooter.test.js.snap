// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <ModalFooter> to render 1`] = `
<ModalFooter
  className="modal-class"
  style={
    Object {
      "width": "100px",
    }
  }
>
  <div
    className="gds-modal__footer -text-right modal-class"
    style={
      Object {
        "width": "100px",
      }
    }
  >
    <div>
      My modal content
    </div>
  </div>
</ModalFooter>
`;
